pkg/db/models/prow.go

```diff
@@ -114,3 +115,21 @@ type Bug struct {
 	Tests          []Test    `gorm:"many2many:bug_tests;"`
 	Jobs           []ProwJob `gorm:"many2many:bug_jobs;"`
 }
+
+// ProwPullRequest represents a GitHub pull request.
+type ProwPullRequest struct {
```

Comment:
Eventually I'd like to drop the release pull requests table, and have them point to this one.  This will let us click through from a release payload to see the job run history on a PR!

Reply:
+1